apps/web/pages/api/book/recurring-event.ts

```diff
@@ -53,16 +54,25 @@ async function handler(req: NextApiRequest & { userId?: number }, res: NextApiRe
       ...booking,
       appsStatus,
       allRecurringDates,
+      thirdPartyRecurringEventId,
       currentRecurringIndex: key,
       noEmail: key !== 0,
     };
 
     const eachRecurringBooking = await handleNewBooking(recurringEventReq, {
       isNotAnApiCall: true,
-      skipAvailabilityCheck: key >= numSlotsToCheckForAvailability,
+      skipAvailabilityCheck: key >= numSlotsToCheckForAvailability || thirdPartyRecurringEventId !== "",
     });
-
     createdBookings.push(eachRecurringBooking);
+
+    if (thirdPartyRecurringEventId == "") {
+      if (eachRecurringBooking.references!.length > 0) {
+        const bookingReference = eachRecurringBooking.references![0];
```

Comment:
I found the issue: google_calendar is not always the first reference. And in my case daily_video is. That's why it never got the `recurringMeetingId`

Reply:
@CarinaWolli I've fixed it, can you check if it works now?